packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx

```diff
@@ -95,18 +108,21 @@ const EditProfileForm = (props: FormikProps<ProfileValues>) => {
         <FormTextInput name='bio' label='Bio' multiline maxLength={256} />
         <FormTextInput name='location' label='Location' />
         <FormTextInput
+          editable={!isTikTokVerified}
```

Comment:
awesome additions!

Reply:
i personally prefer "readOnly" as the prop name, there is some precedent for this name, heres MUI: 
<img width="1194" alt="image" src="https://user-images.githubusercontent.com/8230000/220451482-d9a62ea2-39d4-47a4-8740-7a3a49eafa4f.png">

Reply:
oh jk, i see editable as a prop for text-input, okay ignore all this and the one below

Reply:
readOnly is also a prop for TextInput though, I could change it to that

Reply:
no worries lets stay with react-native api

Reply:
It is! https://reactnative.dev/docs/textinput#readonly

Reply:
oh wait, im so confused lmao